apps/workflow_editor/views/App.js

```diff
@@ -6,35 +6,24 @@ import React, { Component } from 'react';
 
 import AppContainer from 'common-web-ui/views/AppContainer';
 
+import { navigation } from '../config/routes';
+
 export default class App extends Component {
 
   state = {
-    title: this.title
+    navigation
   };
 
   render() {
+    document.body.style.overflow = 'hidden';
     return (
       <AppContainer
           ref="container"
-          className="app"
-          header={null}
-          title={this.state.title}
-          children={this.props.children}
-          styles={{content: {padding: 0}}} />
+          disableAppBar={true}
+          disableTabPadding={true}
+          navigation={this.state.navigation}
+          {...this.props} />
```

Comment:
'spread/rest operator' is only available in ES6 (use esnext option).<br>Possible strict violation.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (83% scanned).